google-cloud-bigtable/.rubocop.yml

```diff
@@ -0,0 +1,53 @@
+AllCops:
+  Exclude:
+    - "google-cloud-bigtable.gemspec"
+    - "lib/google/bigtable/v2/**/*"
+    - "lib/google/cloud/bigtable/v2/**/*"
```

Comment:
The build is broken with a Rubocop error: "23 files inspected, 495 offenses detected".

The source file paths include: `lib/google/cloud/bigtable/v2.rb` and `
lib/google/cloud/bigtable/credentials.rb`, and `
lib/google/cloud/bigtable/admin/v2/`.

 I think the list of exclusions needs to be updated to exclude all source files. (Or the generated code needs to conform to Rubocop rules.)

Reply:
I just excluded all of `google/cloud/...` since this is all autogen. (There are two issues in toolkit tracking the "right" fix, which is to get the codegen to conform to the style rules, googleapis/toolkit#1120, googleapis/toolkit#1536.)